iceoryx_hoofs/include/iceoryx_hoofs/design_pattern/builder.hpp

```diff
@@ -43,19 +43,24 @@
 ///     // END
 ///   };
 /// @endcode
-// NOLINTJUSTIFICATION brackets around macro parameter would lead in this case to compile time failures
+// AXIVION Next Construct AutosarC++19_03-A16.0.1 this macro is used to prevent a large amount of boilerplate code
+// which cannot be realized with templates or constexpr functions
+// AXIVION Next Construct AutosarC++19_03-A16.0.6 brackets around macro parameter would lead in this case to compile
+// time failures
+// AXIVION Next Construct AutosarC++19_03-A16.3.1 false positive, only double '##' is used but no single '#'
```

Comment:
```suggestion
// AXIVION Next Construct AutosarC++19_03-M16.3.1 false positive, only double '##' is used but no single '#'
```

I think you need to update the justification, too. Both `#` and `##` are not recommend in a single macro.

Reply:
@mossmaurice okay, I misunderstood this. I thought only one in the sense of either or. So only one, either `#` or `##` is allowed